src/styles/blocks/stars.scss

```
@@ -0,0 +1,28 @@
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background: url(../images/star.svg) no-repeat center;
+    background-size: auto;
+    margin-right: 4px;
+    display: block;
+  }
+}
+
+.stars--1 .stars__star,
+.stars--2 .stars__star,
+.stars--3 .stars__star,
+.stars--4 .stars__star,
+.stars--5 .stars__star {
+  background-image: url(./../images/star-active.svg);
+}
+
+.stars--1 .stars__star:nth-child(n + 2),
+.stars--2 .stars__star:nth-child(n + 3),
+.stars--3 .stars__star:nth-child(n + 4),
+.stars--4 .stars__star:nth-child(n + 5) {
+  background-image: url(./../images/star.svg);
```

```suggestion
.stars--1 .stars__star:nth-child(-n + 1) 
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url(./../images/star-active.svg);
```
set active star only when you needed to avoid duplication
or at least you could set active star in `&__star` and set non-active where it's needed as you did